packages/common/src/utils/submittedDataUtils.ts

```diff
@@ -0,0 +1,85 @@
+import { functions } from '@overturebio-stack/lectern-client';
+import {
+	SchemaData,
+	SchemaValidationError,
+	SchemasDictionary,
+} from '@overturebio-stack/lectern-client/lib/schema-entities.js';
+import { groupBy, has } from 'lodash-es';
+import { Dependencies } from '../config/config.js';
+import { NewSubmittedData } from '../models/submitted_data.js';
+
+const utils = (dependencies: Dependencies) => {
+	const LOG_MODULE = 'SUBMITTED_DATA_UTILS';
+	const { logger } = dependencies;
+	return {
+		/**
+		 * Creates a map of SubmittedData grouped by entities
+		 * @param {Array<NewSubmittedData>} data
+		 * @returns
+		 */
+		mapSchemaDataByEntity: (
```

Comment:
hmm I'd argue rather than declaring the function's implementation details, the function's name should describe what it takes as input and what the outcome is. thus `getDataByEntityName` sounds more appropriate to me, tbh.

Reply:
changed this function name to `groupSchemaDataByEntityName`.  the full purpose of this function is to return a list of Schema data grouped by EntityName

Reply:
perfect! naming things is rather hard, so great choice!